app/forms/flood_risk_engine/steps/base_form.rb

```diff
@@ -44,6 +38,15 @@ def view_path
       # to over ride the header box and define its own styling
       attr_reader :no_header_in_show
 
+      # Default is to render a view/partial whose name == step, but for some reason step
+      # is a separate local and not part of the Form, so we cannot return it as a default here.
+      # Leave it to the show view to determine whether to render default or a diff partial
+      attr_reader :partial_to_render
+
+      def partial_to_render?
+        !partial_to_render.nil?
```

Comment:
Personally I'd use the built-in Rails #to_partial_path on the form, so in show.html.erb you can do something like
```
<%= render form, form: form, step: step, f: f %>
```
and you don't need `form.partial_to_render?` instead in the form you could do 
```
```